seedlet/inc/jetpack.php

```diff
@@ -14,9 +14,6 @@
  * See: https://jetpack.com/support/responsive-videos/
  */
 function seedlet_jetpack_setup() {
-	// Add theme support for Responsive Videos.
-	add_theme_support( 'jetpack-responsive-videos' );
```

Comment:
What other impact does removing this line have? Is there a reason we want to remove support for responsive videos? (https://jetpack.com/support/responsive-videos/)

Reply:
Varia is missing the `jetpack-responsive-videos` support and videos seem to be sized correctly and maintain their responsiveness.

I'm assuming this was needed in older themes that didn't account for responsive videos.

Reply:
Yeah I did a bit more digging and found this: https://github.com/Automattic/jetpack/issues/10678#issuecomment-497023187

Looks like its safe to remove.

Reply:
Great, thanks for double-checking. I will also remove it from TT1.